ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeViewshed/AnalyzeViewshed.cpp

```diff
@@ -162,65 +162,64 @@ void AnalyzeViewshed::calculateViewshed()
   // Assign a physical location to the new point feature based upon where the user clicked on the map view
   inputFeature->setGeometry(m_inputOverlay->graphics()->at(0)->geometry());
 
-  // connect to addFeature status changed signal
-  connect(inputFeatures, &FeatureCollectionTable::addFeatureCompleted, this, [this, inputFeatures](const QUuid&, bool success)
```

Comment:
I have a question here, `addFeatureAsync()` does not return success/failure.
Is this intentional change in the API?
The TaskWatcher version of signal is this - (`void addFeatureCompleted(QUuid taskId, bool success);`)

Reply:
That is the nice thing about the QFuture workflow. If you get into your .then handler, it succeeded.

For the bool==false, case you can hookup on `onFailed` handler if you want to handle that. In this case, it simply exits early. You don't need to do that anymore here.